pkg/session_recording/recordings.jsx

```diff
@@ -340,7 +349,7 @@
          * Set the cockpit location to point to the specified recording.
          */
         navigateToRecording(recording) {
-            cockpit.location.go([recording.id]);
+            cockpit.location.go([], $.extend(cockpit.location.options, { id: recording.id }));
```

Comment:
1. For example - user sends a link to admin with id and filters - admin can now go back to the same filtering and see, maybe, another recordings related in some way to this filter.
2. If we send only id, then it's only possible to go back only without filters - because URL won't have this state and we will store state internally only, so if page will be refreshed filters will be lost also.

From perspective of coding option 2 will require some minor changes.